samples/authenticateAPIKey.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2024 Google LLC
```

Comment:
Designed to match:
- https://cloud.google.com/docs/authentication/api-keys#using-with-client-libs